integreat_compass/cms/models/offers/tag.py

```diff
@@ -9,7 +9,7 @@ class Tag(AbstractBaseModel):
     Data model representing a Tag.
     """
 
-    title = models.CharField(
+    tag = models.CharField(
```

Comment:
Why have you decided to rename this? I think title is a good naming, as if we want to add something more later, e.g. category, description or whatever, title would be a appropriate naming. Also now i think "title" is better than "tag".

Reply:
Both the "name" fields of contact and organization, as well as the "title" fields of tag and offer_version were clashing, since all of those values are stored in a single request object.

Reply:
I think what you should use is: https://docs.djangoproject.com/en/4.2/ref/forms/api/#prefixes-for-forms
Please undo this renamings, it should definately not be necessary to have unique attribute names across different models. Maybe this article helps, too: https://collingrady.wordpress.com/2008/02/18/editing-multiple-objects-in-django-with-newforms/

Reply:
Neat, did not know about the prefixes. That looks like a good idea, thanks.